StudentsManager.Spa/src/pages/HomePage.css

```
@@ -0,0 +1,644 @@
+    background: rgba(95, 177, 168, 0.05);
+    border: 1px solid var(--matrix-color);
+    padding: 50px;
+    border-radius: 24px;
+    max-width: 700px;
+    margin: 0 auto;
+}
+
+.challenge-snippet {
+    display: block;
+    font-family: var(--mono-font);
+    color: var(--matrix-color);
```

Stylelint (17.7.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)